cmd/agent/common/system_probe_config_helper.go

```diff
@@ -43,10 +45,15 @@ func SetupSystemProbeConfig(sysProbeConfFilePath string) error {
 		return err
 	}
 
-	// The full path to the location of the unix socket where connections will be accessed
+	// The address where connections will be accessed. This is a path to a unix socket (unix) or a TCP address (windows).
 	// This is not necessarily set in the system-probe.yaml, so set it manually if it is not
-	if !config.Datadog.IsSet("system_probe_config.sysprobe_socket") {
-		config.Datadog.Set("system_probe_config.sysprobe_socket", proc_config.GetSocketPath())
+	if !config.Datadog.IsSet(sysProbeSocketConfig) {
+		config.Datadog.Set(sysProbeSocketConfig, proc_config.GetSocketPath())
+	} else {
+		if err := proc_config.ValidateSysprobeSocket(config.Datadog.GetString(sysProbeSocketConfig)); err != nil {
+			log.Errorf("Could not parse system_probe_config.sysprobe_socket: %s", err)
```

Comment:
if we don't return an error, that indicates that this is recoverable?
If so, log should be a warning?

Reply:
I was matching the log level of other config parsing errors.